.travis.yml

```diff
@@ -22,7 +22,11 @@ before_install:
   - if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then
         openssl aes-256-cbc -K $encrypted_1ef8dfbdb114_key -iv $encrypted_1ef8dfbdb114_iv -in travis.tar.gz.enc -out travis.tar.gz -d;
         tar -xzf travis.tar.gz;
-        export INTEGRATION_TEST_FLAGS="-Dit.spanner=true -Dit.storage=true -Dit.config=true -Dit.pubsub=true -Dit.logging=true";
+        export INTEGRATION_TEST_FLAGS="-Dit.spanner=true -Dit.storage=true -Dit.config=true -Dit.pubsub=true -Dit.logging=true
+            -Dit.jpa-sample=true
+            -Dspring.cloud.gcp.sql.database-name=houses
+            -Dspring.cloud.gcp.sql.instance-connection-name=spring-cloud-gcp-ci:us-central1:testmysql
```

Comment:
I think we should stay consistent and choose one way to pass these in, and I prefer the way we have this in the other integration tests: https://github.com/spring-cloud/spring-cloud-gcp/blob/master/spring-cloud-gcp-data-spanner/src/test/resources/application-test.properties

The downside of this travis file method is that we can't use this if we decide to run the tests from intellij or mvn commandline if we don't also type these out in the command

Reply:
I would be OK with `spring.cloud.gcp.sql.database-name=houses` being in a configuration file, but here I really don't want to mention the CI project name, `spring-cloud-gcp-ci`, in the tests code.

Reply:
My opinion is that it's ok to have that project name / instance-connection-name in an application-test.properties file buried deep in the test directories. I don't think it's any more visible than this travis.yml file at the very top directory :/ 

It's up to you!

Reply:
It's not only about visibility, it's also about the fact that external users won't have access to this project and won't be able to run the test even if they followed the same naming conventions as described in the README.

Reply:
I don't think I follow; isn't what you say true regardless of what method we provide these settings ? (in application-test.properties or travis.yml)

Reply:
This also constitutes an argument to change our other integration tests that use GCP-global config values to use travis.yml to provide those settings, correct? (for example the bucket for storage integration tests). should we make an issue of that then ?

Reply:
Yep we should have an issue for that too.